ccs-scale-infra-shared/terraform/modules/infrastructure/network/vpc.tf

```diff
@@ -153,6 +153,17 @@ resource "aws_network_acl" "scale_external" {
     to_port    = 22
   }
 
+  # Allow inbound traffic on ports 80 for HTTP response
+  # (TODO: Why is this necessary / not on the ephemeral range?)
+  ingress {
+    protocol   = "tcp"
+    rule_no    = 41
+    action     = "allow"
+    cidr_block = data.aws_vpc.scale.cidr_block
```

Comment:
Ah, wonder if we can make the sendgrid port VPC range only - can't remember if I tried that or not, would be better if so..